concepts/temporal-adjusters/introduction.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+When working with `java.time` objects like `LocalDate`, you often need to perform complex date adjustments, such as finding the "second Tuesday of the month". Instead of calculating this manually, you can use **`TemporalAdjusters`**.
+
+The `java.time.temporal.TemporalAdjusters` (note the plural 's') utility class provides static methods for many common date adjustments. You apply these adjustments using the `.with()` method on a temporal object (like `LocalDate`).
+
+```exercism/note
+Using `with(TemporalAdjuster)` returns a _new_ instance and does not update the existing instance, as `java.time` objects are immutable.
+```
+
+```java
+import java.time.LocalDate;
+import java.time.Month;
+import java.time.DayOfWeek;
+import java.time.temporal.TemporalAdjusters;
+
+LocalDate date = LocalDate.of(2024, Month.AUGUST, 15); // 2024-08-15 (a Thursday)
+
+// Use a predefined adjuster: find the last day of the month
+LocalDate lastDay = date.with(TemporalAdjusters.lastDayOfMonth());
+// => 2024-08-31
+```
+
+Here are some key adjusters useful for scheduling calculations:
+
+## Finding Ordinal Day of Week
+
+Use `dayOfWeekInMonth(int ordinal, DayOfWeek dayOfWeek)` to find the nth occurrence of a day of the week.
+
+```java
+// Find the second Tuesday of August 2024
+LocalDate secondTuesday = date.with(TemporalAdjusters.dayOfWeekInMonth(2, DayOfWeek.TUESDAY));
+// => 2024-08-13
+```
+
+## Finding First/Last Day of Week in Month
+
+Use `firstInMonth(DayOfWeek dayOfWeek)` or `lastInMonth(DayOfWeek dayOfWeek)`.
+
+```java
+// Find the last Monday of August 2024
+LocalDate lastMonday = date.with(TemporalAdjusters.lastInMonth(DayOfWeek.MONDAY));
+// => 2024-08-26
+```
+
+## Finding Relative Occurrences ("Teenth" dates)
```

Comment:
I think this is a left over from writing for the "meetup" exercise? Since this is an introduction to the temporal adjusters concept, I think we can copy the _Relative Adjustments (Next/Previous)_ content from the about.md here.